nextflow_schema.json

```diff
@@ -153,6 +162,7 @@
                 "fusioncatcher_ref": {
                     "type": "string",
                     "format": "directory-path",
+                    "exists": true,
```

Comment:
Since we are always fetching the attribute, won't this force user to download references for tools they are not planning to use?